controllers/article.js

```diff
@@ -0,0 +1,52 @@
+import 'babel-polyfill';
+import models from '../models';
+
+const { Article, User, Profile } = models;
+
+/**
+ * @class
+ */
+class Articles {
```

Comment:
This does not need to be a class. It would have been nice if the class has extra details to be added to each static method on it. Since it doesn't, please change to a simple object that has methods on it. What you do in this new file sets the tone for what others will follow.

Reply:
I've made the changes.